.travis.yml

```diff
@@ -172,7 +172,8 @@ jobs:
 
       script:
         # Build conda package
-        - conda build -q conda-recipe --output-folder bld-dir -c defaults -c conda-forge -c paulscherrerinstitute
+        - conda build --debug conda-recipe --output-folder bld-dir -c defaults -c conda-forge -c paulscherrerinstitute
```

Comment:
Is the `--debug` flag the new default?  Or was this an accidental comment?

Reply:
Sneaking in a debug flag to figure out why the mac conda build is failing.